test-clients/src/main/java/com/hedera/services/bdd/suites/consensus/TopicDeleteSuite.java

```diff
@@ -111,7 +111,7 @@ private HapiApiSpec deleteWithAdminKey() {
 	}
 
 	private HapiApiSpec deleteFailedWithWrongKey() {
-		long PAYER_BALANCE = 1_999_999_999_999L;
+		long PAYER_BALANCE = 1_999_999_999L;
```

Comment:
reason to lower balance ?

Reply:
When running all tests together in suite, its giving `INSUFFICIENT_PAYER_BALANCE`. As we are not testing anything in this test with higher balance, I reduced the balance on two tests.